.github/repo_meta.yaml

```diff
@@ -0,0 +1,31 @@
+# point_of_contact: the owner of this repository, can be a GitHub user or GitHub team
+point_of_contact: sfc-gh-mrui sfc-gh-bli sfc-gh-ema sfc-gh-sshankar
+
+# production: whether this repository meets the criteria for being "production", see
+# go/production-repository-criteria for criteria
+#
+# If this is not a production repository, set this field to false and delete
+# the rest of the fields below.
+production: true
+
+# distributed: whether any source code in this repository is distributed directly to customers
+# (e.g. driver and frontend software)
+distributed: true
+
+# modified: whether any open source dependencies in this repository have been modified
+modified: false
+
+# release_branches: list of release branch patterns, exact matches or regex is acceptable
```

Comment:
Hi, Nathan,

Spark connector uses github tags for the release, not branch. How to specify it?
For example, the latest release tags are:
```
v2.10.0-spark_3.0
v2.10.0-spark_3.1
v2.10.0-spark_3.2
```

Reply:
what about
v*-spark_*